### PR TITLE
update imaginary docs to reflect cap_add sys_nice

### PR DESCRIPTION
**Pull request by szaimen**
As it is suggested in https://github.com/nextcloud/server/issues/35643#issuecomment-1455137443

### CODE REVIEW

File: admin_manual/installation/server_tuning.rst [hunk 1]
```diff
@@ -160,6 +160,7 @@ Nextcloud to use Imaginary by editing your `config.php`:
 .. warning::
 
    Make sure to start Imaginary with the `-return-size` command line parameter. Otherwise, there will be a minor performance impact. The flag requires a recent version of Imaginary (newer than v1.2.4) and is by default added to the `aio-imaginary` container.
+   Also make sure to add the capability `SYS_NICE` via `--cap-add=sys_nice` or `cap_add: - SYS_NICE` as it is required by imaginary.
```
Comment (st3iny) [inline admin_manual/installation/server_tuning.rst:163]: Could you elaborate on the reason that this capability is needed? Randomly granting a capability has security implications and admins should be able to at least comprehend it.

E.g. amend the sentence with: `to generate HEIC previews.`

Comment (szaimen) [reply admin_manual/installation/server_tuning.rst]: done 👍

**Review by st3iny — APPROVED** [review]

**Pull request merged**